### DISCUSSION TITLE
V1.2 Link from NAICS Codes to Summary Emission Factors from Previous Version

### DISCUSSION THREAD

**Discussion by smcnichols94** [USEPA/supply-chain-factors#50] [Q&A] [answered]
Hello,

Previously I was using the US EEIO supply chain emissions factors by summary. Since the factors have switched to NAICS codes I am able to match to the detailed industry and commodities, but I don't see a way to map the NAICS codes to the previous summary factors. Is there a way this can be done?

Thanks,
Susan

**Comment by bl-young** [comment, marked as answer]
Producing the underlying USEEIO model will generate a crosswalk between the USEEIO sectors and other levels of aggregation. I have attached a USEEIO (detail) to summary crosswalk here. This will align with BEA's crosswalk.

[USEEIOv2_crosswalk_summary.csv](https://github.com/USEPA/supply-chain-factors/files/12003726/USEEIOv2_crosswalk_summary.csv)

**Reply by smcnichols94** [reply]
Thank you!

**Comment by nliusont** [comment]
Very helpful. Thank you for this. I had the exact same question.

Follow up - is it sufficient to average the v1.2 factors at the NAICS-6 level to get the summary industry factors? Or should this technically be a weighted average based on the share of the summary industry that each NAICS-6 is within the economy?

Edit: Looking at the crosswalk you provided, the USEEIO Reference Codes do not correspond to the same codes provided in the v1.2 Supply Chain Emission Factors. For example, code 111110 is in the v1.2 emission factors but is not in your list of reference codes.

Edit 2: Sorry, I now see the USEEIO Reference Code column in the EF file. Doh!

**Reply by bl-young** [reply]
Because these data are coefficients, in kg / $, it would be more appropriate to weight them rather than just a straight average, especially if consolidating at the summary level. Really the best approach would be to use the available R code to [generate your own version](https://github.com/USEPA/supply-chain-factors/blob/main/model-specs/USEEIOv2.1.19-GHG.yml#L3) of the model at a summary level, if feasible.

**Reply by jbaglia90** [reply]
@bl-young To confirm, in order to generate the "industry summary" factors that used be in previous releases of the published csv release - is the correct approach just to change the following in the underlying "model-specs" file (USEEIOv2.1.19-GHG.yml): 

BaseIOLevel: "Detail" → **“Summary”**
CommodityorIndustryType: "Commodity" → **“Industry”**
DisaggregationSpecs: "WasteDisaggregationDetail" → **null**

**Reply by bl-young** [reply]
Yes, i believe so - you can also change the `IOYear`, because in summary models those are available annually.

**Reply by jbaglia90** [reply]
@bl-young Thank you! Which IO years are available for use with the summary models for the following models? 
- USEEIOv2.1.19-GHG
- USEEIOv2.2.22-GHG

I.e., to be used when generating factors industry summary factors from the following Supply Chain Factor releases?
- v1.3
- v1.2.3

**Reply by bl-young** [reply]
Sorry I've lost track a bit of what you are trying to do. If you want to recreate the released data, but using a summary model, then you should keep the year the same I suppose.

**Reply by jbaglia90** [reply]
Basically I'm trying to generate the most up-to-date supply chain factors by industry summary name level instead of the NAICS level. 

So it sounds like to have more representative factors for more recent activity, when generating the models at a summary level instead of a detail level, I can actually use more recent IO data even if the factors themselves are less granular (i.e., 66 BEA summary industries instead of 1,016 NAICS codes)?

I.e., if trying to generate the most up-to-date industry summary factors, what would I set the IOYear to in the model-specs if I download the R model from here: (https://github.com/USEPA/supply-chain-factors/releases)

Would it be IOYear: 2019?

**Reply by bl-young** [reply]
The supply chain factors are a wrapper on the useeio models. You can find the latest available data in [useeior](https://github.com/USEPA/useeior/releases). For example, there you can see that as of v1.7 we have summary IO data through 2023. The year of the IO data is distinct from the year of the GHG data. The latest GHG data remains 2022.

The USEEIOv2.2.22-GHG model uses the 2022 GHG data, but because it is a detail model it uses IOYear of 2017. If you change to a summary model you can update the IOYear to 2022.

For example, we just recently published [v2.5 models](https://catalog.data.gov/dataset/useeio-v2-5-models). You can see the [USEEIOv2.5-kingbird-22 model](https://dmap-data-commons-ord.s3.amazonaws.com/useeior/USEEIOv2.5-kingbird-22.yml)[^1] is similar to what you are describing. Note that that model uses import emission factors, which to date have not been incorporated into the supply chain factors and so you would likely not want to use this model spec directly.

[^1]: We have recently updated the naming schema, hence the use of "kingbird". See the [description here](https://github.com/USEPA/USEEIO/blob/master/models.md)
